frontend/assets/images/inline-svgs/raw/cross.svg

```diff
@@ -1 +1,8 @@
-<svg width="36" height="36" xmlns="http://www.w3.org/2000/svg"><path d="M0 30.162c.374-1.227 1.766-1.343 2.452-2.284 1.812-2.482 4.468-4.127 6.525-6.365 1.02-1.109 2.269-1.984 3.12-3.269.547-.825.06-2.183-.674-2.835-1.022-.91-2.362-1.367-3.233-2.45-.573-.712-1.483-.889-2.247-1.255-1.386-.665-2.24-1.845-3.097-2.981-.374-.496-.912-.988-.952-1.809-.034-.707.127-1.382.66-1.443.774-.088 1.589-.819 2.444.109 1.229 1.333 2.484 2.644 4.26 3.375.938.387 1.75 1.173 2.549 1.833 1.301 1.074 2.63 2.104 4.007 3.071.481.338.985.315 1.482-.121 2.124-1.867 4.319-3.634 6.749-5.11 1-.607 1.888-1.425 2.744-2.235.806-.762 1.744-1.271 2.697-1.793 1.058-.578 2.033-1.282 2.866-2.167h.972c.852.63 2.262.384 2.676 1.703v.243c-.237 1.619-1.466 2.081-2.771 2.725-2.815 1.389-5.131 3.599-7.806 5.22-1.482.898-2.445 2.383-4.101 3.089-1.008.43-.88 1.574.112 2.249 1.99 1.355 3.659 3.095 5.559 4.548.957.732 1.885 1.502 3.096 1.979.938.37 1.909 1.122 2.698 1.862.651.611 1.477.925 2.096 1.63.761.868 1.258 1.9.631 2.895-.586.928-1.334.979-2.496.76-1.152-.219-1.516-.924-2.101-1.592-.074-.085-.119-.254-.198-.268-2.699-.48-4.134-2.978-6.49-4.074-1.14-.53-2.068-1.704-3.051-2.626-.537-.504-1.08-.824-1.774-.996-.531-.131-.799-.643-1.158-1.005-.885-.892-1.882-1.038-2.766-.087-1.46 1.574-2.854 3.214-4.265 4.834-1.662 1.911-2.855 4.242-4.842 5.85-.946.766-1.986 1.451-2.968 2.195h-.243c-.079-.075-.149-.205-.237-.219-1.149-.181-2.346-.271-2.696-1.714l-.229-.012v-1.46z" fill="#333" clip-rule="evenodd" fill-rule="evenodd"/></svg>
\ No newline at end of file
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Event though these are minified buy the build step it's probably still worth running these through https://jakearchibald.github.io/svgomg/ just to clean out the cruft and maybe remove `fill="#BDBDBD"` just to be on the safe side.